### DISCUSSION TITLE
Z‐Stick Gen5 USB Controller no Z-Wave+ ?!

### DISCUSSION THREAD

**Discussion by hestiap** [zwave-js/zwave-js-ui#1985] [Q&A] [answered]
Hi,
I've a "Z‐Stick Gen5 USB Controller" wich is zwave+ but in the GUI, I could not get it shown as zwave+, even after several Re-interview Node. Most of other nodes are correct?
Is it a problem?

Another question for this node, I could node have a successful "Heal", it always fails

zwavejs2mqtt: 5.12.0
zwave-js: 8.7.6

**Comment by robertsLando** [comment, marked as answer]
In next version (6.0.0) zwave+ column, secuirity, beaming, interview status/heal status and some other columns will be hidden for controller node as they make no sense. So in general it shouldn't be something you have to worry about :)

**Reply by hestiap** [reply]
Thanks for your quick reply :-)
